common/rfb/CSecurityTLS.cxx

```diff
@@ -281,6 +287,9 @@ void CSecurityTLS::setParam()
     if (gnutls_certificate_set_x509_crl_file(cert_cred, X509CRL, GNUTLS_X509_FMT_PEM) < 0)
       vlog.error("Could not load user specified certificate revocation list");
 
+    if (gnutls_certificate_set_x509_key_file (cert_cred, X509CERT, X509KEY, GNUTLS_X509_FMT_PEM) < 0)
+      vlog.error("Could not load user specified client certificate");
```

Comment:
I wonder if we need to have a bit more explicit error handling. What is the user experience if there is some issue with the certificate or key file?